lib/teaspoon/instrumentation.rb

```diff
@@ -45,14 +45,14 @@ def instrumented_response
 
     def self.ignored?(asset)
       Array(Teaspoon::Coverage.configuration.ignore).any? do |ignore|
-        asset.pathname.to_s.match(ignore)
+        asset.filename.match(ignore)
```

Comment:
Is `filename` present in current versions, or is this a breaking change for others?